src/zipAll.js

```diff
@@ -0,0 +1,40 @@
+var _curry1 = require('./internal/_curry1');
+
+/**
+* Creates a new list out of the lists supplied by pairing up
+* equally positioned items from both lists. The returned list is
+* truncated to the length of the shorter of the two input lists.
```

Comment:
it doesn't it takes a list of n lists (matrix).

This is a transpose function for matrices, but for ramda naming convention its named zipAll
:+1: